### PR TITLE
profile:

### PR DESCRIPTION
**Pull request by nandanasreeraj123**
This PR solves issue #

**What does this PR do?**

This creates the profile page for the application

### CODE REVIEW

File: src/components/profile/profilebox.jsx [hunk 1]
```diff
@@ -0,0 +1,354 @@
...
+            <div className="langButtonCSS-below">
+              <p>CSS</p>
+            </div>
+          </div>
+        </div>
+        <div className="savebutton">
+          <p>Save</p>
+        </div>
+      </div>
+      <style jsx global>
```
Comment (irshadjsr21) [inline src/components/profile/profilebox.jsx:157]: Please add this to a CSS file and import it as a module. For details checkout PR #5.

Comment (nandanasreeraj123) [reply src/components/profile/profilebox.jsx]: I have added the same and rectified it. Please have a look.

Comment (irshadjsr21) [reply src/components/profile/profilebox.jsx]: Nice work

Comment (nandanasreeraj123) [reply src/components/profile/profilebox.jsx]: Thank you :)

**Pull request merged**